api/data/NMBS/liveboard.php

```diff
@@ -114,22 +114,21 @@ private static function parseData($xml, $time, $lang, $fast = false)
             $journey = $data->Journey[$i];
 
             $left = 0;
+            $cancelled = false;
             $delay = (string) $journey['delay'];
             if ($delay == '-') {
                 $delay = '0';
             }
-
-            $platform = '';
-            if (isset($journey['platform'])) {
-                $platform = (string) $journey['platform'];
+            
+            if ($delay == 'cancel') {
+                $delay = 999999; // Indicate something is wrong if `cancelled` is not read by the client
```

Comment:
I find this not a good solution... Without upgrading code, the GUIs will now show a delay of 999999. I believe we should just keep this 0?